src/components/scenes/SendScene2.tsx

```diff
@@ -1130,6 +1136,10 @@ const SendComponent = (props: Props): React.ReactElement => {
       try {
         if (beforeTransaction != null) await beforeTransaction()
       } catch (e: unknown) {
+        console.error(
+          'Error from before transaction route param hook: ',
```

Comment:
Why are we not surfacing this error to the UI?

Reply:
Because it fails gracefully. This was the behavior. I have no idea why.